charts/dvpe-deployment-gloo/templates/gloo_upstream.yaml

```diff
@@ -8,13 +8,23 @@ kind: Upstream
 metadata:
   name: {{ $upstreamName }}
   namespace: {{ .Values.gloo.upstream.namespace }}
+  {{- if .Values.gloo.fds.enabled }}
+  labels:
+    discovery.solo.io/function_discovery: enabled
+  {{- end }}
 spec:
   kube:
     selector:
       app: {{ $serviceName }}
     serviceName: {{ $serviceName }}-svc
     serviceNamespace: {{ .Release.Namespace }}
     servicePort: {{ .Values.service.spec.ports.https.port }}
+    {{- if .Values.gloo.fds.enabled }}
+    serviceSpec:
+      rest:
+        swaggerInfo:
+          url: http://{{ $serviceName }}-svc.{{ .Release.Namespace }}:{{ .Values.service.spec.ports.https.port }}{{ .Values.gloo.fds.path }}
```

Comment:
I would move this to the helpers.tpl
I think it is the reason why the evaluation is not successful

Reply:
Hi @christodo ,

The pipeline has failed because the kubeval script does not provide error handling for empty variables.
I did not bump the version string in the Chart.yaml file, therefor the kubeval script did not recognized changes and leaves a variable empty which caused into an error in helm & kubeval check command.